src/controllers/authController.js

```diff
@@ -0,0 +1,176 @@
+const User = require('../models/user.js');
+const userService = require('../services/userService.js');
+const tokenService = require('../services/tokenService.js');
+const jwtService = require('../services/jwtService.js');
+const ApiError = require('../exeptions/apiError.js');
+const bcrypt = require('bcrypt');
+
+const validateEmail = (value) => {
+  if (!value) {
+    return 'Email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(value)) {
+    return 'Email is not valid';
+  }
+};
+
+const validaPassword = (value) => {
+  if (!value) {
+    return 'Password is required';
+  }
+
+  if (value.length < 6) {
+    return 'At least 6 characters';
+  }
+};
+
+const register = async (req, res) => {
+  const { name, email, password } = req.body;
+
+  const errors = {
+    email: validateEmail(email),
+    password: validaPassword(password),
```

Comment:
The function `validaPassword` is called here, but it should be `validatePassword` due to the typo in the function name.